### PR TITLE
V3

### PR DESCRIPTION
**Pull request by 3y3**

### CODE REVIEW

File: demo/src/Components/DocPage/index.scss [hunk 1]
```diff
@@ -0,0 +1,6 @@
+//@import "@gravity-ui/uikit/styles/styles.css";
```
Comment (martyanovandrey) [inline demo/src/Components/DocPage/index.scss:1]: forgotten comments

File: demo/src/Components/DocPage/index.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+    useEffect(() => {
+        updateBodyClassName(theme);
+    }, [theme]);
+
+    const props = {
+        ...getContent(lang, singlePage),
+        vcsType,
+        router,
+        // headerHeight: fullScreen ? 0 : 64,
+        // headerHeight: fullScreen ? 0 : 64,
```
Comment (martyanovandrey) [inline demo/src/Components/DocPage/index.stories.tsx:212]: few forgotten comments in this file

File: src/components/DocPage/DocPage.tsx [hunk 3]
```diff
@@ -1,45 +1,48 @@
...
+import {DocLayout} from '../DocLayout';
+import {DocPageTitle} from '../DocPageTitle';
+import {Feedback, FeedbackView} from '../Feedback';
+import {HTML} from '../HTML';
+import {MiniToc} from '../MiniToc';
+import {SearchBar, withHighlightedSearchWords} from '../SearchBar';
+import {TocNavPanel} from '../TocNavPanel';
 
-import LinkIcon from '../../../assets/icons/link.svg';
+// TODO(V3): replace with gravity-ui
```
Comment (martyanovandrey) [inline src/components/DocPage/DocPage.tsx:39]: maybe replace with this icon ?

import {Link} from '@gravity-ui/icons';

File: package.json [hunk 4]
```diff
@@ -1,56 +1,77 @@
...
-    "build:compile": "tsc -p tsconfig.publish.json",
-    "build": "npm run build:clean && npm run build:compile && npm run build:copy",
-    "storybook:start": "start-storybook -p 7008",
-    "storybook:build": "build-storybook -c .storybook -o storybook-static",
+    "build:compile": "./esbuild/build.js",
+    "_build:declarations:esm": "tsc --emitDeclarationOnly -p tsconfig.esm.json",
+    "_build:declarations:cjs": "tsc --emitDeclarationOnly -p tsconfig.cjs.json",
+    "build": "run-s build:clean build:compile _build:declarations:*",
+    "_build:watch": "./esbuild/build.js --watch",
+    "_storybook:watch": "cd demo && npm run storybook",
```
Comment (martyanovandrey) [inline package.json:67]: there is no "storybook" in package.json

File: README.md
Comment (martyanovandrey) [inline README.md:1]: The way the storybook is launched has changed


> To start the development server with storybook run the following:
> 
> ```
> cd demo
> npm install
> npm run dev
> ```

**Review by martyanovandrey — APPROVED** [review]

File: V3.md [hunk 5]
```diff
@@ -0,0 +1,170 @@
+## Breaking
+
+- Update to `@gravity-ui/uilit@5`
```
Comment (Kyzyl-ool) [inline V3.md:3]: uikit

File: package.json (hunk 4 above)
Comment (Kyzyl-ool) [reply package.json:67]: you need to run `npm ci` in `demo folder`. maybe one should add "postinstall" script?

**Pull request merged**